### PR TITLE
fix: Add platform-specific congrats messages

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Description of changes

As called out in #2043, we want to update the messaging after automated tests find no issues. This fix updates the congratulations message in Web, Android, and reports. It adds a deps value that gets passed into the class that generates the message, so that the default value can be overridden. I went with the override approach because the report package doesn't currently pass in the necessary props. There's probably a future PR in the reports package to enable this, but since there was a stated goal to time-box this fix, I used default + override.

Part of the fix was updating the word-break property to break properly on word boundaries.

#### Pull request checklist

- [x] Addresses an existing issue: #2043 
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [x] (UI changes only) Added screenshots/GIFs to description above
- [] (UI changes only) Verified usability with NVDA/JAWS

Congrats message in web automated results:
image

Congrats message in generated reports:
image

Congrats message in unified (clipped to display just the message):
image

### CODE REVIEW

File: src/electron/views/renderer-initializer.ts [hunk 1]
```diff
@@ -292,6 +292,8 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then(
             unifiedResultToIssueFilingDataConverter: null, // we don't support issue filing right now
             windowUtils: null,
             setFocusVisibility,
+            customCongratsMessage:
+                "No failed automated checks were found. Continue investigating your app's accessibility compliance through manual testing",
```
Comment (smoralesd) [inline src/electron/views/renderer-initializer.ts]: is there a missing `.` at the end of the string?

File: src/reports/components/report-sections/no-failed-instances-congrats.tsx [hunk 2]
```diff
@@ -5,16 +5,30 @@ import * as React from 'react';
...
+export type NoFailedInstancesCongratsProps = {
+    deps: NoFailedInstancesCongratsDeps;
+};
+
+export const NoFailedInstancesCongrats = NamedFC<NoFailedInstancesCongratsProps>(
+    'NoFailedInstancesCongrats',
+    props => {
+        const message = props.deps.customCongratsMessage
+            ? props.deps.customCongratsMessage
+            : "No failed automated checks were found. Continue investigating your website's accessibility compliance through manual testing using Tab stops and Assessment in Accessibility Insights for Web";
```
Comment (smoralesd) [inline src/reports/components/report-sections/no-failed-instances-congrats.tsx]: would it be better to pass this text (the "coda" of the congrats message for AI-Web) as a deps the same as we do for AI-Android?
Doing so makes all the cases follow the same pattern, which is easier to understand.

Comment (DaveTryon) [reply src/reports/components/report-sections/no-failed-instances-congrats.tsx]: The reports always pass in a null value, so it's not really feasible to use a code there. Given the messages, I'm not sure how much would be the "base" and how much would be the "coda". My inclination is to stick with this for now, then (after we update the reports to pass in a value), follow a pattern where we have no default and the string is completely specified by the caller. Happy to chat more about this if you want.

File: src/electron/views/renderer-initializer.ts (hunk 1 above)
Comment (DaveTryon) [reply src/electron/views/renderer-initializer.ts]: Yes. I'l add it to both strings.

File: src/reports/components/report-sections/no-failed-instances-congrats.tsx [hunk 3]
```diff
@@ -5,16 +5,30 @@ import * as React from 'react';
...
+};
+
+export type NoFailedInstancesCongratsProps = {
+    deps: NoFailedInstancesCongratsDeps;
+};
+
+export const NoFailedInstancesCongrats = NamedFC<NoFailedInstancesCongratsProps>(
+    'NoFailedInstancesCongrats',
+    props => {
+        const message = props.deps.customCongratsMessage
```
Comment (karanbirsingh) [inline src/reports/components/report-sections/no-failed-instances-congrats.tsx]: you might be able to use ?? here instead of ternary

Comment (DaveTryon) [reply src/reports/components/report-sections/no-failed-instances-congrats.tsx]: Will do this when I fix the merge conflict

**Pull request merged**